twingly_search/query.py

```diff
@@ -24,77 +28,146 @@ class Query(object):
                    Assumes UTC if the datetime object has no timezone set
     """
 
-    DATETIME_FORMAT = "%Y-%m-%d %H:%M:%S"
-
     def __init__(self, client):
         """
         No need to call this method manually, instead use Client#query.
 
         :param client: the client that this query should be connected to
         """
         self.client = client
-        self.pattern = ''
-        self.language = ''
-        self._start_time = None
-        self._end_time = None
+        self.search_query = ''
+        self._language = ''
+        self._start_date = None
+        self._end_date = None
+
+    @property
+    def start_date(self):
```

Comment:
@roback Maybe you have to deprecate them also in Ruby client?

It's just quite weird for me, as a developer, that I'm going to set "time", but do actually set time and date. And as it will be converted to `start-date: [value]` probably it should just follow the documentation as is, so that it would be easier to match what Client do with the search query language.